extensions/llama/python/llama.py

```diff
@@ -95,25 +105,43 @@ async def _run_inference_helper(self, model_input, options) -> List[Output]:
             except TypeError:
                 texts = [response["choices"][0]["text"]]
 
+            output_data_value : str = ''
+            last_response = texts[-1]
+            if isinstance(last_response, str):
+                output_data_value = last_response
+            else:
+                raise ValueError(f"Output {last_response} needs to be of type 'str' but is of type: {type(last_response)}")
```

Comment:
Instead of this, should we concatenate all the strings in `texts`? Isn't that the complete response?

Reply:
Or is this similar to `choices` in openai inference, where a model can return `n` different responses if asked to do so (in which case we construct a single `ExecuteResult` per choice.

Reply:
> in which case we construct a single ExecuteResult per choice

Yea this is what flagged in https://github.com/lastmile-ai/aiconfig/issues/630, I'll add this issue to the code comment